data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            link: "https://www.conservation.org/",
+            keywords: ["conservation", "ngo"],
+            label: "conservation",
+        },
+        description: {
+            content: "Conservation International (CI) is an established NGO protecting land, sea, wildlife and sustainable relationships between human and nature worldwide."
+            + "  CI works with local communities, governments and businesses to conserve as much nature as possible.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        actions: {
+            content: "- [Protecting biodiversity hotspots](https://www.conservation.org/priorities/biodiversity-hotspots)"
+            + "\n- [Field projects protecting land, animal and plant species](https://www.conservation.org/priorities/field-projects)"
+            + "\n- [Environmental Advocacy with governments](https://www.conservation.org/priorities/working-with-governments)"
+            + "\n- [Fighting wildlife trafficking](https://www.conservation.org/priorities/wildlife-trade-and-trafficking)"
+            + "All [topics can be found online](https://www.conservation.org/priorities).",
+            updatedBy: "Alexander", updatedAt: "2020-02-16",
+        },
+        yearlyIncome: {
+            content: "- [149M $ in 2018](https://www.conservation.org/about/annual-report)" +
+                "\n- [158M $ in 2017](https://www.conservation.org/docs/default-source/publication-pdfs/fy17_annual_report.pdf?sfvrsn=e3d11069_3)" +
+                "\n- [212M $ in 2016](https://www.conservation.org/docs/default-source/publication-pdfs/2016-annual-report.pdf?sfvrsn=b268b345_3)" +
+                "\n- [165M $ in 2015](https://www.conservation.org/docs/default-source/publication-pdfs/2016-annual-report.pdf?sfvrsn=b268b345_3)",
+            updatedBy: "Alexander", updatedAt: "2019-02-16"
+        },
+        established: {
+            content: "[1987](https://www.conservation.org/about]",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        employees: {
+            content: "[53](https://www.conservation.org/docs/default-source/publication-pdfs/fy18_annual_report.pdf).",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "World Business Council for Sustainable Development",
+            link: "https://www.wbcsd.org/",
+            keywords: ["private-sector", "corporate", "network", "think-tank"],
+            label: "wbcsd",
+        },
+        description: {
+            content: "WBCSD is a network of 200 of the largest cooperations (8.5T $ combined revenue, 19M employees)."
+            + " It includes food multinationals as well as large petrochemical corporations."
+            + " Its stated mission is a sustainable economy until 2050. The economic interests of its membership make it seem unlikely that WBCSD can be a pivotal force in that endeavor."
```

Comment:
leave critisism out

Reply:
Happy to discuss and tone down. I think if we make suggestions, we should have a clear stance.

Reply:
I am absolutely wit you. Would change the wording into 
```suggestion
            + " Its stated mission is a sustainable economy until 2050. The economic interests of its membership have to been taken into account when it comes to review this organization.```